app/experimenter/experiments/bugzilla.py

```diff
@@ -162,3 +159,14 @@ def create_experiment_bug(experiment):
 def get_bugzilla_id(bug_url):
     query = urlparse(bug_url).query
     return int(parse_qs(query)["id"][0])
+
+
+def set_bugzilla_id_value(bug_url):
+    if "show_bug.cgi?id=" in bug_url:
+        try:
+            data_science_bug_id = get_bugzilla_id(bug_url)
+            if bug_exists(data_science_bug_id):
+                return [data_science_bug_id]
+        except ValueError:
+            return None
```

Comment:
Both of these `return None` are redundant since a function will always return None.  Also I think this try/except is also unnecessary, I would change the get_bugzilla_id to `.get("id", [None])[0]` so that it doesn't raise at all and then just check if it's None before you make the bug_exists call.